translations/en.json

```diff
@@ -301,7 +301,9 @@
     "verificationResult": "Verification result",
     "verified": {
       "title": "Parking cards added",
-      "text": "Your parking cards email **{{email}}** has been added. You can see your cards in the Parking cards section.",
+      "text_empty": "Your parking cards email **{{email}}** has been added. You can see your cards in the Parking cards section.",
+      "text": "Your parking cards email **{{email}}** has been added. You can see your cards in the Parking cards section.\n\n Vehicle with license plate **{{licencePlates}}** added as well. You can manage your vehicles in the section Vehicles.",
+      "text_plural": "Your parking cards email **{{email}}** has been added. You can see your cards in the Parking cards section.\n\n Vehicles with license plates **{{licencePlates}}** added as well. You can manage your vehicles in the section Vehicles.",
```

Comment:
It would be nice to separate the "changing" part to other translation key to prevent duplication.